plugins/modules/elb_network_lb.py

```diff
@@ -108,6 +108,13 @@
       - This parameter is mutually exclusive with I(subnet_mappings).
     type: list
     elements: str
+  security_groups:
+    description:
+      - A list of the names or IDs of the security groups to assign to the load balancer.
+      - Required if I(state=present).
+      - If C([]), the VPC's default security group will be used.
+    type: list
+    elements: str
```

Comment:
```suggestion
  security_groups:
    version_added: 7.3.0
    description:
      - A list of the names or IDs of the security groups to assign to the load balancer.
      - Required if I(state=present).
      - If C([]), the VPC's default security group will be used.
    type: list
    elements: str
```

`version_added` is required, because it is a new parameter.